lib.php

```diff
@@ -141,8 +141,8 @@ function theme_boost_union_get_pre_scss($theme) {
 
     // Prepend variables first.
     foreach ($configurable as $configkey => $targets) {
-        $value = isset($theme->settings->{$configkey}) ? $theme->settings->{$configkey} : null;
-        if (empty($value)) {
+        $value = get_config('theme_boost_union', $configkey);
+        if (!($value)) {
```

Comment:
The goal of this PR is to replace `$theme->settings->{$configkey}` with `get_config('theme_boost_union', $configkey)`.
This is fine. However, as this part of the function is duplicated from upstream Boost Core and to avoid any hickups with edge cases, I would like to ask you to keep the rest of the login in these two lines unchanged unless you prove me that it must be changed.

Reply:
I'll change this like I did in #262 .

Reply:
Thank you :)